testing/trino-benchto-benchmarks/README.md

```diff
@@ -48,8 +52,10 @@ data-sources:
 environment:
   name: TRINO-DEV
 
-presto:
+trino:
```

Comment:
You have to also either change the data source name to `presto` or update all benchmarks.

Reply:
Guess I'll have to return to the old name. We can do the rename in a separate commit/PR